core/src/i18n/pl.yaml

```
@@ -12,12 +12,12 @@ brokers:
   title: Brokerzy
 catalog:
   menu-title: Katalog
-  subtitle: Wzbogać swoje doświadczenie dodatkowymi usługami, które są instalowane poza klastrem.
+  subtitle: Rozszerz możliwości za pomocą dodatkowych usług instalowanych poza klastrem.
   title: Katalog Serwisów
 cluster-addons:
   title: Dodatki do klastra
 cluster-role-bindings:
-  title: Powiązania ról klastra
+  title: Powiązania Ról klastra
```

```suggestion
  title: Powiązania ról klastra
```
I noticed you don't capitalize the proper names in Polish (which is OK if that's what we choose to go with), so let's be consistent here.